onsset/onsset.py

```diff
@@ -574,76 +596,61 @@ def grid_penalties(self):
 
         logging.info('Classify road dist')
         #define bins as -inf to 5, 5 to 10, 10 to 25, 25 to 50, 50 to inf
-        road_distance_bins = [float("-inf"),5,10,25,50,float("inf")]
+        road_distance_bins = [0,5,10,25,50,float("inf")]
         #define classifiers
         road_distance_labels = [5,4,3,2,1]
 
-        self.df[SET_ROAD_DIST_CLASSIFIED] = pd.cut(self.df[SET_ROAD_DIST], road_distance_bins,
-                                            labels=road_distance_labels).astype(float)
+        ROAD_DIST_CLASSIFIED = pd.cut(data_frame[SET_ROAD_DIST], road_distance_bins,
+                                            labels=road_distance_labels, include_lowest=True).astype(float)
 
         logging.info('Classify substation dist')
-        #define bins as -inf to 0.5, 0.5 to 1, 1 to 5, 5 to 10, 10 to inf
-        substation_distance_bins = [float("-inf"),0.5,1,5,10,float("inf")]
+        #define bins as 0 to 0.5, 0.5 to 1, 1 to 5, 5 to 10, 10 to inf
+        substation_distance_bins = [0,0.5,1,5,10,float("inf")]
         #define classifiers
         substation_distance_labels = [5,4,3,2,1]
 
-        self.df[SET_SUBSTATION_DIST_CLASSIFIED] = pd.cut(self.df[SET_SUBSTATION_DIST], substation_distance_bins,
+        SUBSTATION_DIST_CLASSIFIED = pd.cut(data_frame[SET_SUBSTATION_DIST], substation_distance_bins,
```

Comment:
Create a function for this classification method as I have demonstrated for the road classification.